homeassistant/components/camera/__init__.py

```diff
@@ -500,3 +541,29 @@ def _write_image(to_file, image_data):
             _write_image, snapshot_file, image)
     except OSError as err:
         _LOGGER.error("Can't write image to file: %s", err)
+
+
+async def async_handle_play_stream_service(camera, service):
+    """Handle play stream services calls."""
+    from homeassistant.components.media_player.const import (
+        ATTR_MEDIA_CONTENT_ID, ATTR_MEDIA_CONTENT_TYPE,
+        SERVICE_PLAY_MEDIA, DOMAIN as DOMAIN_MP)
+    from homeassistant.components.stream import async_request_stream
+    hass = camera.hass
+    try:
+        fmt = service.data.get(ATTR_FORMAT)
+        entity_ids = service.data.get(ATTR_MEDIA_PLAYER)
+
+        url = await async_request_stream(hass, camera.stream_source, fmt=fmt)
+        data = {
+            ATTR_MEDIA_CONTENT_ID: url,
+            ATTR_MEDIA_CONTENT_TYPE: 'application/vnd.apple.mpegurl'
+        }
+
+        if entity_ids:
+            data[ATTR_ENTITY_ID] = entity_ids
+
+        await hass.services.async_call(
+            DOMAIN_MP, SERVICE_PLAY_MEDIA, data, blocking=True)
+    except HomeAssistantError:
+        pass
```

Comment:
Remove this `try/except`, service is allowing throw exception now. Even you want to catch exception, you should log it here, no silently swallow it.